impeller/entity/contents/content_context.h

```diff
@@ -337,6 +337,7 @@ class ContentContext {
   /// @brief  Creates a new texture of size `texture_size` and calls
   ///         `subpass_callback` with a `RenderPass` for drawing to the texture.
   std::shared_ptr<Texture> MakeSubpass(ISize texture_size,
+                                       bool enable_msaa,
```

Comment:
Maybe use an enum so this can be self-documenting? Like `kMSAAEnabled`, `kMSAADisabled`.

Reply:
Removed this param.